UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -44,6 +45,10 @@ public partial class UndertaleRoomEditor : DataUserControl
 
         public static readonly PropertyInfo visualOffProp = typeof(Canvas).GetProperty("VisualOffset", BindingFlags.NonPublic | BindingFlags.Instance);
         private static readonly MainWindow mainWindow = Application.Current.MainWindow as MainWindow;
+        private static readonly Regex endNumberEx = new(@"\d+$", RegexOptions.Compiled);
```

Comment:
Personally don't like how this is named.

Reply:
But how else to call it?

Reply:
Something akin to `trailingNumbersPattern`?

Reply:
Yeah, then it will be `trailingNumberRegex`.